### ISSUE TITLE
[Deepcast] Selected text as source doesn't seem to work

### ISSUE THREAD

**Issue by mattouzalem** [raycast/extensions#9077]
### Extension

https://www.raycast.com/mooxl/deepcast

### Description

Hey,

Even if I choose `Selected` as the Preferred Source in the extension settings, selected text doesn't appear in Deepcast.
Please note that it's working well with other Raycast Extensions I use with this setting (WordReference, Quicklinks etc.).

Thanks a lot for the support!

### Steps To Reproduce

1. Select text
2. Trigger Deepcast using a hotkey

### Current Behaviour

No text appears in the extension.

### Expected Behaviour

Selected text shall appear in the extension.

**Comment by erics118** [comment]
This currently currently isn't supported because it would fallback to the clipboard if the selected text is empty, leading to the form to always be filled with some data.

**Comment by mattouzalem** [comment]
Understood, thanks for this info!

**Comment by erics118** [comment]
How about just implementing the selected text part, without defaulting to clipboard? I think that's a solution that makes sense.
You can always easily cmd-v to paste your clipboard into the form.

**Comment by mattouzalem** [comment]
I agree with this solution, it would make it faster to get any text selected from the screen on the extension.

**Comment by erics118** [comment]
@raycastbot reopen this issue

**Comment by erics118** [comment]
I've implemented it in #9083

**Comment by mattouzalem** [comment]
Cool, thanks!!
